lib/developer_portal/lib/liquid/drops/api_spec.rb

```diff
@@ -21,6 +21,13 @@ def service
         return unless (service = @spec.service)
         Drops::Service.new(service)
       end
+
+      desc 'Returns the public url of the service of the spec if it has any or `nil` otherwise'
+      def endpoint
```

Comment:
The description is kinda clear but the name isn't. We have many "endpoint" and the name should let obvious which one it is.
For example `service_proxy_endpoint` (which would be easy with the delegation 😄 )

Reply:
That's actually _the_ endpoint of the API product. Perhaps it's the other "endpoints" that require better names 🙂 

@3scale/system-ux, how would you like to call it?

Reply:
A few options:
- `production_endpoint`
- `production_service_endpoint`
- `production_api_endpoint`
- `production_api_product_endpoint`
- `api_product_endpoint`

Reply:
For me it's ok `service_proxy_endpoint`, but I don't know the names of other endpoints

Reply:
I haven't seen your list, then I like `api_product_endpoint`

Reply:
We went with `api_product_production_public_base_url` after suggestion from @thomasmaas.